server/views/pages/bookedtoday/arrivals/confirmArrival/checkAnswersForCreateNewRecord.njk

```diff
@@ -0,0 +1,72 @@
+{% extends "../../../../partials/layout.njk" %}
```

Comment:
the new page

Reply:
summary-list has not been extracted into a Nunjucks macro because the two pages on which it is used, have slightly different content